spoti-vote-backend/README.md

```diff
@@ -1,6 +1,6 @@
 # OAuth bridge template
 
-This service logs in to Spotify and redirects the user to a given frontend application with a valid access_token as a parameter in the url.
+This service is responsible for handling the logic which affects multiple users.
```

Comment:
text is being displayed in the inner read.me (not the one on the main page of the repo), u sure bout this?

Reply:
well, its the readme for the server and the content is also for the server so it fits, the main readme can also be adapted